### PR TITLE
host-ctr: bump dependencies

### PR DESCRIPTION
**Pull request by etungsten**
**Issue number:**
N/A

**Description of changes:**
Bumps host-ctr all dependencies except for containerd.

**Testing done:**
- Successfully built aws-k8s-1.17/x86_64 AMI and launched instance
- Saw both host-containers@admin and host-containers@control come up successfully.

<details>

```
[ec2-user@ip-123-123-123-123 ~]$ sudo sheltie
bash-5.0# systemctl status host-containers@admin
● host-containers@admin.service - Host container: admin
     Loaded: loaded (/x86_64-bottlerocket-linux-gnu/sys-root/usr/lib/systemd/system/host-containers@.serv
ice; enabled; vendor preset: enabled)
     Active: active (running) since Tue 2020-11-10 20:59:40 UTC; 2min 2s ago
    Process: 3025 ExecStartPre=/usr/bin/mkdir -m 1777 -p ${LOCAL_DIR}/host-containers/admin (code=exited,
 status=0/SUCCESS)
   Main PID: 3059 (host-ctr)
      Tasks: 18 (limit: 9185)
     Memory: 41.0M
     CGroup: /system.slice/system-host\x2dcontainers.slice/host-containers@admin.service
             └─3059 /usr/bin/host-ctr -ctr-id=admin -source=328549459982.dkr.ecr.us-west-2.amazonaws.com/bo
ttlerocket-admin:v0.5.2 -superpowered=true

Nov 10 20:59:45 ip-192-168-25-105.us-west-2.compute.internal host-ctr[3059]: time="2020-11-10T20:59:45Z" 
level=info msg=Unpacking... img="ecr.aws/arn:aws:ecr:us-west-2:328549459982:repository/bottlerocket-admin
:v0.5.2"
Nov 10 21:00:00 ip-192-168-25-105.us-west-2.compute.internal host-ctr[3059]: time="2020-11-10T21:00:00Z" 
level=info msg="Tagging image" imageName="328549459982.dkr.ecr.us-west-2.amazonaws.com/bottlerocket-admin
:v0.5.2"
Nov 10 21:00:00 ip-192-168-25-105.us-west-2.compute.internal host-ctr[3059]: time="2020-11-10T21:00:00Z" 
level=info msg="No clean up necessary, proceeding" ctr-id=admin
Nov 10 21:00:01 ip-192-168-25-105.us-west-2.compute.internal host-ctr[3059]: time="2020-11-10T21:00:01Z" 
level=info msg="Successfully started container task"
Nov 10 21:00:01 ip-192-168-25-105.us-west-2.compute.internal host-ctr[3059]:   % Total    % Received % Xf
erd  Average Speed   Time    Time     Time  Current
Nov 10 21:00:01 ip-192-168-25-105.us-west-2.compute.internal host-ctr[3059]:                             
     Dload  Upload   Total   Spent    Left  Speed
Nov 10 21:00:01 ip-192-168-25-105.us-west-2.compute.internal host-ctr[3059]: [158B blob data]
Nov 10 21:00:01 ip-192-168-25-105.us-west-2.compute.internal host-ctr[3059]: Server listening on 0.0.0.0 
port 22.
Nov 10 21:00:01 ip-192-168-25-105.us-west-2.compute.internal host-ctr[3059]: Server listening on :: port 
22.
Nov 10 21:01:24 ip-192-168-25-105.us-west-2.compute.internal host-ctr[3059]: Accepted publickey for ec2-u
ser from 205.251.233.176 port 32162 ssh2: RSA SHA256:+qAlN881Y/8Rza0+mOmjviCFvfvNWhvHX5q/F+myiSE
bash-5.0# systemctl status host-containers@control
● host-containers@control.service - Host container: control
     Loaded: loaded (/x86_64-bottlerocket-linux-gnu/sys-root/usr/lib/systemd/system/host-containers@.serv
ice; enabled; vendor preset: enabled)
     Active: active (running) since Tue 2020-11-10 20:59:40 UTC; 2min 5s ago
    Process: 3042 ExecStartPre=/usr/bin/mkdir -m 1777 -p ${LOCAL_DIR}/host-containers/control (code=exite
d, status=0/SUCCESS)
   Main PID: 3062 (host-ctr)
      Tasks: 20 (limit: 9185)
     Memory: 47.3M
     CGroup: /system.slice/system-host\x2dcontainers.slice/host-containers@control.service
             └─3062 /usr/bin/host-ctr -ctr-id=control -source=328549459982.dkr.ecr.us-west-2.amazonaws.com/
bottlerocket-control:v0.4.1 -superpowered=false

Nov 10 21:00:41 ip-192-168-25-105.us-west-2.compute.internal host-ctr[3062]: 2020-11-10 21:00:41 INFO [St
artupProcessor] Unable to open serial port /dev/hvc0: open /dev/hvc0: no such file or directory
Nov 10 21:00:41 ip-192-168-25-105.us-west-2.compute.internal host-ctr[3062]: 2020-11-10 21:00:41 ERROR [S
tartupProcessor] Error opening serial port: open /dev/hvc0: no such file or directory
Nov 10 21:00:41 ip-192-168-25-105.us-west-2.compute.internal host-ctr[3062]: 2020-11-10 21:00:41 ERROR [S
tartupProcessor] Error opening serial port: open /dev/hvc0: no such file or directory. Retrying in 5 seco
nds...
Nov 10 21:00:46 ip-192-168-25-105.us-west-2.compute.internal host-ctr[3062]: 2020-11-10 21:00:46 INFO [St
artupProcessor] Unable to open serial port /dev/ttyS0: open /dev/ttyS0: no such file or directory
Nov 10 21:00:46 ip-192-168-25-105.us-west-2.compute.internal host-ctr[3062]: 2020-11-10 21:00:46 INFO [St
artupProcessor] Attempting to use different port (PV): /dev/hvc0
Nov 10 21:00:46 ip-192-168-25-105.us-west-2.compute.internal host-ctr[3062]: 2020-11-10 21:00:46 INFO [St
artupProcessor] Unable to open serial port /dev/hvc0: open /dev/hvc0: no such file or directory
Nov 10 21:00:46 ip-192-168-25-105.us-west-2.compute.internal host-ctr[3062]: 2020-11-10 21:00:46 ERROR [S
tartupProcessor] Error opening serial port: open /dev/hvc0: no such file or directory
Nov 10 21:00:46 ip-192-168-25-105.us-west-2.compute.internal host-ctr[3062]: 2020-11-10 21:00:46 ERROR [S
tartupProcessor] Error opening serial port: open /dev/hvc0: no such file or directory. Retrying in 5 seco
nds...
Nov 10 21:00:51 ip-192-168-25-105.us-west-2.compute.internal host-ctr[3062]: 2020-11-10 21:00:51 ERROR [S
tartupProcessor] Error occurred while opening serial port: Timeout: Serial port is in use or not availabl
e
Nov 10 21:00:51 ip-192-168-25-105.us-west-2.compute.internal host-ctr[3062]: 2020-11-10 21:00:51 ERROR Er
ror occurred during startup of processor: Timeout: Serial port is in use or not available
```

</details>

- Was able to start a ssm session with the launched instance

</details>

**Terms of contribution:**

By submitting this pull request, I agree that this contribution is dual-licensed under the terms of both the Apache License, version 2.0, and the MIT license.

### CODE REVIEW

File: sources/host-ctr/go.mod [hunk 1]
```diff
@@ -3,27 +3,32 @@ module host-ctr
 go 1.12
 
 require (
-	github.com/aws/aws-sdk-go v1.34.3 // indirect
-	github.com/awslabs/amazon-ecr-containerd-resolver v0.0.0-20200804232255-8c110f606e9e
-	github.com/containerd/cgroups v0.0.0-20200710171044-318312a37340 // indirect
+	github.com/Microsoft/go-winio v0.4.15 // indirect
+	github.com/Microsoft/hcsshim v0.8.10 // indirect
```
Comment (samuelkarp) [inline sources/host-ctr/go.mod:7]: If you run `go mod tidy`, does it trim these dependencies?

Comment (etungsten) [reply sources/host-ctr/go.mod:7]: No, it doesn't

Comment (etungsten) [reply sources/host-ctr/go.mod:7]: `go mod why`s
```
$ go mod why github.com/Microsoft/go-winio
# github.com/Microsoft/go-winio
host-ctr/cmd/host-ctr
github.com/containerd/containerd/cio
github.com/Microsoft/go-winio
$ go mod why github.com/Microsoft/hcsshim
# github.com/Microsoft/hcsshim
host-ctr/cmd/host-ctr
github.com/containerd/containerd
github.com/containerd/containerd/archive
github.com/Microsoft/hcsshim

```

Comment (jahkeup) [reply sources/host-ctr/go.mod:7]: That's what I suspected - I'm not leery of these dependencies lingering in this case.

**Review by jahkeup — APPROVED** [review]
Bumps look good to me.

File: sources/host-ctr/go.mod (hunk 1 above)
Comment (samuelkarp) [reply sources/host-ctr/go.mod:7]: I was hoping they'd be trimmed since they're Windows-specific, but oh well...

**Review by samuelkarp — APPROVED** [review]

**Review by zmrow — APPROVED** [review]
:microphone:

**Pull request merged**